grails-app/domain/org/pih/warehouse/invoice/InvoiceItem.groovy

```diff
@@ -146,34 +144,6 @@ class InvoiceItem implements Serializable {
         return orderAdjustment ? orderAdjustment.description : product?.name
     }
 
-    /**
-     * Total shipment item value
-     * @deprecated From now should rely on the amount field instead always calculating it
-     */
-    def getTotalAmount() {
```

Comment:
This is being used in `InvoiceService.getInvoiceItemsCsv`. You should hopefully just need to switch to use amount instead of totalAmount

Reply:
Yup, @alannadolny please replace it in the service as well